test/libraries/cublas/level2.jl

```diff
@@ -72,6 +76,16 @@ k = 13
                     y[i] = alpha * A[i] * x[i] + beta * y[i]
                     @test y[i] ≈ hy
                 end
+                dy = CuArray{elty, 1}[]
+                for i=1:length(A)
+                    push!(dy, CuArray(y[i]))
+                end
+                CUBLAS.gemv_batched!(elty <: Real ? 'T' : 'C', alpha, dA, dy, beta, dx)
+                for i=1:length(A)
```

Comment:
Previous bug: This line was `for i = 1:size(A, 3)`, where `A` is a `Vector` of `CuMatrix`. 

Not sure if that is related to illegal memory access.